node_test.go

```diff
@@ -1312,3 +1315,130 @@ func TestNode_DeleteMultiple_Query(t *testing.T) {
 	}), ".", []interface{}{}))
 	t.Run("RootNodeUnknown", deleteMultipleTest(dasel.New(false), ".", map[string]interface{}{}))
 }
+
+// TestNode_NewFromFile tests parsing from file
+//
+// Not sure what to test here:
+// should all file/parser tests be duplicated?
+// E.g.: TestLoadFromFile, TestNewReadParserFromString
```

Comment:
I'd do a couple of file/parser variations with `TestLoadFromFile`, and then ensure `NewReadParserFromString` tests all variations

Reply:
I've added more permutations of valid file/parsers and covered all `Read` parsers with no-existing file tests.